### PR TITLE
Simply LOTR joke

### LINKED ISSUE #50: Add Lord of the Rings Dad Responses
I.e. "one does not simply walk", "ring", "eye", see" and Dad will respond with some gif from LOTR

### PR DESCRIPTION
**Pull request by king2b3**
Dad now looks for "simply" "one doesn't" or "one does not" to respond with a LOTR quote (closes #50)
Added myself (Bayley) to the authors page
Updated the util to allow for the new simply directory
Updated and fixed spelling mistakes in the README
  Yes we should be scared that I, Bayley, found and corrected spelling mistakes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +19,15 @@ This is meant to be a custom cog to be added to a [Red DiscordBot](https://githu
 ## Adding a New Joke
 + New jokes can usually be added as a new sub-class of the Joke class in the jokes directory.
 + The easiest way to sublcass the Joke object is to look at the existing jokes, but take care to
- 1. Change the `super().__init__("new_joke_name", joke_chance)` to the name of the joke and the default chance of occurance [0.0,100.0]
+ 1. Change the `super().__init__("new_joke_name", joke_chance)` to the name of the joke and the default chance of occurrence [0.0,100.0]
```
Comment (I-question-this) [inline README.md:22]: I appreciate this spelling check.

File: AUTHORS.md [hunk 2]
```diff
@@ -1 +1,2 @@
 - Westland, Tyler <[i-question-this](https://github.com/i-question-this)>
+- King, Bayley <[king2b3](https://github.com/king2b3)>
```
Comment (I-question-this) [inline AUTHORS.md:2]: Yes, perfect. Give credit where blame is due.

File: README.md [hunk 3]
```diff
@@ -18,15 +19,15 @@ This is meant to be a custom cog to be added to a [Red DiscordBot](https://githu
...
  2. Implement the joke within the `_make_joke` method.
  3. Edit `jokes/__init__.py` to import your joke and add an instance of it to the `JOKES` dictionary
   - The proper key for the new instance is "JokeNameJoke"
  4. Add a short description of this joke to this README.
 
 ### Why This Structure?
-The purpose is to have a standarized method of turning a user's message into a joke. 
+The purpose is to have a standardized method of turning a user's message into a joke.
```
Comment (I-question-this) [inline README.md:29]: I appreciate this spelling check. X 2

File: README.md [hunk 4]
```diff
@@ -18,15 +19,15 @@ This is meant to be a custom cog to be added to a [Red DiscordBot](https://githu
...
  4. Add a short description of this joke to this README.
 
 ### Why This Structure?
-The purpose is to have a standarized method of turning a user's message into a joke. 
+The purpose is to have a standardized method of turning a user's message into a joke. 
 Dad does this by randomizing the list of jokes, and then iterating through each joke until either he can make a joke with the message or he runs out of jokes.
 This can be seen in ``on_message`` method in `dad.py`.
 Further the `Joke` class adds the to the list of options that can be set for each guild.
-This allows guilds to modify the probablity that a particular joke will occur, without extra code needing to be written for that.
+This allows guilds to modify the probability that a particular joke will occur, without extra code needing to be written for that.
```
Comment (I-question-this) [inline README.md:33]: I appreciate this spelling check. X 3

File: README.md [hunk 5]
```diff
@@ -6,6 +6,7 @@ This is meant to be a custom cog to be added to a [Red DiscordBot](https://githu
 + Being a programmer is hard -> programmer, I barely know her!
 + In general Dad's jokes are amazing -> General Dad + a gif of a person saluting
 + Dad's jokes are a smashing good time -> A gif of Nigel Thornberries face plastered on someone elses body
++ One does not simply push one past Dad -> A gif from LOTR will serve as a simple reminder
```
Comment (I-question-this) [inline README.md:9]: I appreciate so much that you added this part. Pro developer move.

File: README.md (hunk 4 above)
Comment (king2b3) [reply README.md:33]: You've got to love the VS code spell check plug in I have. actually it sucks

File: README.md (hunk 5 above)
Comment (king2b3) [reply README.md:9]: Gotta start somewhere, documentation is key

File: jokes/simply.py [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            # Log joke
+            LOG.info(f"Simply: {match}")
+            # Construct our response
+            response = {}
+            # Pick random gif
+            simply_gif = random_image(SIMPLY_DIR)
+            # Construct embed
+            embed = discord.Embed.from_dict(response)
+            embed.set_image(url=f"attachment://{simply_gif.filename}")
+            # Send embed and smashing gif
```
Comment (I-question-this) [inline jokes/simply.py:62]: This is a simple gif, and not a smashing gif. -10/10

**Review by I-question-this — CHANGES_REQUESTED** [review]
This is very good code, however it will never run. You will need to edit the jokes/__init__.py file to add this to the list of jokes for Dad to run.

**Comment by I-question-this** [comment]
The proper way to do more changes is to add more commits to the branch that is being merged.

**Review by I-question-this — APPROVED** [review]
Tested this code on my end, and it works beautifully.

**Pull request merged**